reproman/support/exceptions.py

```diff
@@ -197,3 +197,22 @@ class SSHAccessDeniedViaAuthKeys(BaseException):
     """
     def __init__(self, user):
         self.msg = "access for user '%s' denied via authorized_keys" % user
+
+
+#
+# Run related exceptions
+#
+class JobError(RuntimeError):
+    """reproman run  execution related error.
+    """
+    def __init__(self, *args, failed=None):
+        super().__init__(*args)
```

Comment:
Why match/propagate `RuntimeError`s `args` but not its `kwargs`?

Reply:
it seems that it takes no kwargs:
```
In [3]: RuntimeError(buga="duga")                                                                                
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-825a24e4a3fd> in <module>
----> 1 RuntimeError(buga="duga")

TypeError: RuntimeError() takes no keyword arguments
```
so I will keep as is

Reply:
Thank you for checking.  When inspecting objects with `pydoc`, I should remember to check for the second line below and not trust the signature if I see it, though in this case I don't think its suggestion is useful.

```
 |  __init__(self, /, *args, **kwargs)
 |      Initialize self.  See help(type(self)) for accurate signature.
```